cat/CaT-service.yaml

```diff
@@ -3310,32 +3377,67 @@ components:
           type: string
           readOnly: true
         description: 
-          type: string 
-          
-    QuestionSummary:
+          type: string       
+ 
+    Question:
       description: >-
-        This is summary of the questions within a Question Group
+        A question that must be answered by the supplier as part of a tender or a question answered by the buyer to provide information to support the supplier.
       type: object
       properties:
-        id: 
-          type: string
-        title: 
-          type: string 
-        description: 
-          type: string
-        dataType:
-          type: string         
- 
-    Question:
-      $ref: 'https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-api-definitions-common/main/CCS_OCDS_Standards/CCS-OCDS_Schema.yaml#/components/schemas/Requirement'
-      #description: >-
-      #  Generic term for an event put out to the market. Each is an optional procurement step in the procurement process. This call shows the current status of each in the procurement process.
-      #type: array
-      #items:
-      #  type: object
-      #  properties:
-      #    id: 
-      #      type: string 
+        non-OCDS:
```

Comment:
Can we make this `nonOCDS` for consistency with the camelCase property naming strategy?